### PR TITLE
Create Korea.sqf

### PR DESCRIPTION
**Pull request by HeroesMaker**
Added south korean army loadout, need Broma armory for the K2 rifle.

### CODE REVIEW

**Comment by Auri3l** [comment]
Does this require "_rifleList" to be added to loadouts for ammo?

**Comment by Auri3l** [comment]
Oh shit hello BromA

**Comment by HeroesMaker** [comment]
nah, i just used the R3F mag because they are centered correctly in the rifle model but it can use the rhs/niarms/vanilla stanags too

File: loadouts/factions/Korea.sqf [hunk 1]
```diff
@@ -0,0 +1,261 @@
+// INFO ========================================================================
+/*
+   South korean army loadout.
+*/
+
+_factionID = "ROKA";
+_factionName = "ROK Army";
+_factionStructure = "HIGH-TIER-CCO";
```
Comment (Coryf88) [inline loadouts/factions/Korea.sqf:8]: `"HIGH-TIER-CCO"` doesn't exist anymore. Change to "HIGH-TIER" or another one.

File: loadouts/factions/Korea.sqf [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+_weaponsAA = _STINGER;
+_weaponsAT = _JAVELIN;
+_commonSMG = _RHSM4;
+_commonRCO = "BRM_K2CS";
+_marksmanSCOPE = "rhsusf_acc_m8541_mrds";
+_specATSCOPE = "rhs_optic_maaws";
+_commonCCO = "BRM_K2CQS";
+_commonEOT = "rhsusf_acc_eotech_552";
+_commonRAIL = "rhsusf_acc_anpeq15side_bk";
+_commonBIPOD = "rhsusf_acc_tdstubby_blk";
```
Comment (Coryf88) [inline loadouts/factions/Korea.sqf:60]: Is incompatible with all of the weapons:
	`_commonRIFLE`
	`_reconRIFLE`
	`_commonRIFLEGL`
	`_commonAR`
	`_commonMARKSMAN`
	`_commonMG`

File: loadouts/factions/Korea.sqf [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+_commonMAGNIFIED = "rhsusf_acc_nxs_3515x50f1_h58";
+_commonSUPPRESSOR = "rhsusf_acc_rotex5_grey";
+_commonPISTOLSUPPRESSOR = "";
+_NVG = "rhsusf_ANPVS_14";
+
+_mgSCOPE = _commonEOT;
```
Comment (Coryf88) [inline loadouts/factions/Korea.sqf:66]: Is incompatible with `_commonAR`.

File: loadouts/factions/Korea.sqf [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+_specAT = _GUSTAV;
+_weaponsAA = _STINGER;
+_weaponsAT = _JAVELIN;
+_commonSMG = _RHSM4;
+_commonRCO = "BRM_K2CS";
+_marksmanSCOPE = "rhsusf_acc_m8541_mrds";
+_specATSCOPE = "rhs_optic_maaws";
+_commonCCO = "BRM_K2CQS";
+_commonEOT = "rhsusf_acc_eotech_552";
+_commonRAIL = "rhsusf_acc_anpeq15side_bk";
```
Comment (Coryf88) [inline loadouts/factions/Korea.sqf:59]: Only compatible with `_reconRIFLE` not:
	`_commonRIFLE`
	`_commonRIFLEGL`
	`_commonAR`
	`_commonMARKSMAN`
	`_commonMG`

If intended, click resolve conversation.

**Review by Coryf88 — CHANGES_REQUESTED** [review]

File: loadouts/factions/Korea.sqf (hunk 4 above)
Comment (HeroesMaker) [reply loadouts/factions/Korea.sqf:59]: this one is intended.

**Pull request merged**